Sources/CgRPC/shim/cgrpc.h

```diff
@@ -118,6 +118,31 @@ typedef struct grpc_event {
   void *tag;
 } grpc_event;
 
+typedef enum grpc_arg_type {
+  GRPC_ARG_STRING,
+  GRPC_ARG_INTEGER,
+  GRPC_ARG_POINTER
+} grpc_arg_type;
+
+typedef struct grpc_arg_pointer_vtable {
+  void* (*copy)(void* p);
```

Comment:
Nit: Would you mind having the asterisk after the space? I.e. ` void *`, not `void* `. (Also below.)